src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -558,7 +558,13 @@ export class NavbarComponent implements OnInit, OnDestroy {
                 } else if (this.lastRouteUrlSegment === 'code-editor' && segment === 'new') {
                     // - This route is bogus and needs to be replaced in the future, display no crumb
                     break;
-                } else if (this.lastRouteUrlSegment === 'programming-exercises' && segment === 'import') {
+                } else if (
+                    (this.lastRouteUrlSegment === 'programming-exercises' ||
+                        this.lastRouteUrlSegment === 'text-exercises' ||
+                        this.lastRouteUrlSegment === 'modeling-exercises' ||
+                        this.lastRouteUrlSegment === 'file-upload-exercises') &&
+                    segment === 'import'
+                ) {
```

Comment:
Will this work? (What about quizzes? They are currently not part of this condition?)
```suggestion
                } else if (this.lastRouteUrlSegment.endsWith('-exercises') && segment === 'import') {
```

Reply:
Thanks for the suggestion.
unfortunately, it doesn't work and no breadcrumbs are displayed.